libvast/vast/address_synopsis.hpp

```diff
@@ -90,6 +90,11 @@ class buffered_address_synopsis : public synopsis {
     ips_.insert(*addr_view);
   }
 
+  size_t size() const override {
+    return sizeof(buffered_address_synopsis)
+      + ips_.size()*sizeof(typename decltype(ips_)::node_type);
```

Comment:
For span, we have a `span::size_bytes` function that explicitly returns the size in bytes. I think this and all the other new functions should be named `size_bytes` as well.